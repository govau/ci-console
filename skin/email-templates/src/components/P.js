import React from "react";
import styled from "styled-components";

const P = styled.p`
  margin: 10px 0;
  padding: 0;
  mso-line-height-rule: exactly;
  -ms-text-size-adjust: 100%;
  -webkit-text-size-adjust: 100%;
  color: ${({ theme }) => theme.textColor};
  font-family: ${({ theme }) => theme.fontFamily};
  font-size: 16px;
  line-height: 150%;
  text-align: left;
`;

export default ({ children }) => <P>{children}</P>;
