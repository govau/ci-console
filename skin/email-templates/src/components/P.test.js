import React from "react";
import renderer from "react-test-renderer";

import P from "./P";

test("renders without crashing", () => {
  const tree = renderer.create(<P>content</P>);
  expect(tree).toMatchSnapshot();
});
