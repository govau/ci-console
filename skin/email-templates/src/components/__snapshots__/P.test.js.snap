// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
.c0 {
  margin: 10px 0;
  padding: 0;
  mso-line-height-rule: exactly;
  -ms-text-size-adjust: 100%;
  -webkit-text-size-adjust: 100%;
  font-size: 16px;
  line-height: 150%;
  text-align: left;
}

<p
  className="c0"
>
  content
</p>
`;
